packages/story-editor/src/components/keyboardShortcutsMenu/keyboardShortcutList.js

```diff
@@ -365,6 +365,25 @@ const shortcuts = {
             </kbd>
           ),
         },
+        {
+          label: __('Incremental nudge', 'web-stories'),
+          shortcut: (
+            <kbd>
+              <TranslateWithMarkup
+                mapping={{
+                  lkbd: <LargeKey />,
+                  shortcut: <Shortcut shortcut="shift" />,
+                }}
+              >
+                {sprintf(
+                  /* translators: 1: Shift key. */
+                  __('<lkbd>%s</lkbd> <span>+ arrows</span>', 'web-stories'),
```

Comment:
We should probably use arrow icons for consistency as elsewhere.

Reply:
using arrows but it will probably make the design look bad I have attached the screenshot for the same in the PR description.

Reply:
@amovar18 Thanks for the screenshot, will get some UX feedback and get back to you ASAP.

Reply:
I think the term "arrow keys" makes the most sense. The text there isn't too jarring.

Reply:
Any thoughts on the "incremental nudge" wording? Without context it seems hard to grasp.

Reply:
@swissspidy should the wording be somewhat like `Move element by 1px`?

Reply:
@amovar18 It's still being confirmed, sorry for the delay.

Reply:
@miina any updates?

Reply:
@amovar18 Still waiting (will send another reminder).

Reply:
https://github.com/google/web-stories-wp/pull/10029#issuecomment-998490111 any thoughts on this?

Reply:
@amovar18 Let's go with what [Andrew recommended](https://github.com/google/web-stories-wp/pull/10029#discussion_r777498221): 

> I think the term "arrow keys" makes the most sense. The text there isn't too jarring.

Reply:
@amovar18 Got the final copy from UX now:
"Move element 1px"